pom.xml

```diff
@@ -251,6 +251,11 @@
         <artifactId>pinot-hadoop</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Do you need to also include the ingestion-common?

Reply:
No need, pinot-hadoop, pinot-spark will include it.